Sources/VaporJWT/ConfigError.swift

```diff
@@ -0,0 +1,26 @@
+/// Errors that can take place during the config
+/// of required JWT entities such as the signer
+public struct ConfigError: Swift.Error {
+    var code: Int
```

Comment:
I agree with `let` (the `var` is accidental carry over from trying out a computed prop)

As for `public`, I see the code, in this error pattern, being an implementation detail. If we make it public then changing which code belongs to which error becomes an implementation detail. Really all the code serves for right now is to allow `==` to function.